apps/ui/src/views/Space/Settings.vue

```diff
@@ -286,43 +303,48 @@ watchEffect(() => setTitle(`Edit settings - ${props.space.name}`));
 </script>
 
 <template>
-  <div class="!pb-0">
-    <UiScrollerHorizontal class="sticky z-40" with-buttons gradient="xxl">
-      <div class="flex px-4 space-x-3 bg-skin-bg border-b min-w-max">
-        <AppLink
-          v-for="tab in tabs.filter(tab => tab.visible)"
-          :key="tab.id"
-          :to="{
-            name: 'space-settings',
-            params: { space: route.params.space, tab: tab.id }
-          }"
-          type="button"
-          class="scroll-mx-8"
-          @focus="handleTabFocus"
-        >
-          <UiLink :is-active="tab.id === activeTab" :text="tab.name" />
-        </AppLink>
-      </div>
-    </UiScrollerHorizontal>
+  <UiScrollerHorizontal
+    class="sticky z-40 top-[72px]"
+    with-buttons
+    gradient="xxl"
+  >
+    <div class="flex px-4 space-x-3 bg-skin-bg border-b min-w-max">
+      <AppLink
+        v-for="tab in tabs.filter(tab => tab.visible)"
+        :key="tab.id"
+        :to="{
+          name: 'space-settings',
+          params: { space: route.params.space, tab: tab.id }
+        }"
+        type="button"
+        class="scroll-mx-8"
+        @focus="handleTabFocus"
+      >
+        <UiLink :is-active="tab.id === activeTab" :text="tab.name" />
+      </AppLink>
+    </div>
+  </UiScrollerHorizontal>
+  <div
+    v-bind="$attrs"
+    class="!h-auto"
+    :style="`min-height: calc(100vh - ${bottomToolbarHeight + 114}px)`"
+  >
     <div v-if="loading" class="p-4">
       <UiLoading />
     </div>
     <div
       v-else
-      class="space-y-4 flex flex-col pb-[100px] h-full"
-      :class="{
-        'mx-4 max-w-[592px]': !['profile', 'whitelabel'].includes(activeTab)
-      }"
-      style="min-height: calc(100vh - 114px)"
+      class="flex-grow"
+      :class="{ 'px-4 pt-4': activeTab !== 'profile' }"
     >
-      <div v-show="activeTab === 'profile'" class="flex-grow pb-[100px]">
+      <template v-if="activeTab === 'profile'">
```

Comment:
v-if can be applied directly on `FormSpaceProfile`, and avoid the `template` tag